frontend/tests/README.md

```diff
@@ -228,6 +228,47 @@ Before running any tests, start with [mise en place](../README.md#mise-en-place)
 
       ![Playwright Inspector](https://user-images.githubusercontent.com/6309723/196775284-81bd4853-b4c6-45c4-827b-22e97b97ee06.png)
 
+## Troubleshooting
+
+### preSetup Access Token
+
+If you see an issue that looks like this
+```
+  ✘  1 [preSetup] › tests/common/playwright.global.preSetup.ts:11:7 › global preSetup › Get access token (0ms)
+Error: ENOENT: no such file or directory, open '/var/folders/l5/ygnys3jj7n9f12p826j9448c0000gq/T/playwright-transform-cache-503/f7/context_f76b0cdd8a23a6ac47b3cccbc8fed32460f74ab0.js'
+
+   at playwright.config.ts:11
+
+   9 | import fs from "fs";
+  10 | import { LOGIN_STATE_FILENAME } from "tests/common/constants";
+> 11 | import { COMMON_PLAYWRIGHT_CONTEXT } from "tests/common/context";
+     | ^
+  12 | import { getFeatureFlags } from "tests/common/featureFlags";
+  13 | import { SKIP_LOGIN } from "tests/common/constants";
+  14 | import { shouldUseRdevToken } from "tests/utils/helpers";
+```
+You can resolve it by just removing the folder
+```
+rm -rf /var/folders/l5/ygnys3jj7n9f12p826j9448c0000gq/T/playwright-transform-cache-503
+```
+
+### AWS Region Missing in Setup
+
+If you see an issue within the Setup that looks like this
+```
+  ✘  2 [setup] › tests/common/playwright.global.setup.ts:10:7 › global setup › login (13.3s)
+🔐🪵 Logging in...
+Error: Region is missing
+```
+
+You can resolve this by updating your AWS config file to include the following snippet. On a Linux/Mac, this should be in `~/.aws/config`.
+```
+[default]
+role_arn       = arn:aws:iam::699936264352:role/poweruser
```

Comment:
Not sure if we should redact this for security? I see Github does that in the lint error output 😆 

```
[default]
 role_arn       = arn:aws:iam::***:role/poweruser // <--- THIS
Error: Process completed with exit code 1.
```

Reply:
ah good call.. will do!